tuxemon/core/client.py

```diff
@@ -38,6 +41,11 @@
 
 logger = logging.getLogger(__name__)
 
+#Rich Presence Implementation Part 1
+client_id = '824398106182811698'  # Put your Client ID here, this is mine lol
```

Comment:
Possibly needs changed have not tested with an separate account

Reply:
lets move this to the config file

Reply:
how is the secret handled?  can we move that to the config as well?